packages/Webkul/Paypal/src/Http/Controllers/SmartButtonController.php

```diff
@@ -127,28 +127,28 @@ protected function buildRequestBody()
             'purchase_units' => [
                 [
                     'amount'   => [
-                        'value'         => (float) $cart->sub_total + $cart->tax_total + ($cart->selected_shipping_rate ? $cart->selected_shipping_rate->price : 0) - $cart->discount_amount,
+                        'value'         => $this->smartButton->formatCurrencyValue((float) $cart->sub_total + $cart->tax_total + ($cart->selected_shipping_rate ? $cart->selected_shipping_rate->price : 0) - $cart->discount_amount),
                         'currency_code' => $cart->cart_currency_code,
 
                         'breakdown'     => [
                             'item_total' => [
                                 'currency_code' => $cart->cart_currency_code,
-                                'value'         => (float) $cart->sub_total,
+                                'value'         => $this->smartButton->formatCurrencyValue((float) $cart->sub_total),
                             ],
 
                             'shipping'   => [
                                 'currency_code' => $cart->cart_currency_code,
-                                'value'         => (float) ($cart->selected_shipping_rate ? $cart->selected_shipping_rate->price : 0),
+                                'value'         => $this->smartButton->formatCurrencyValue((float) ($cart->selected_shipping_rate ? $cart->selected_shipping_rate->price : 0)),
                             ],
 
                             'tax_total'  => [
                                 'currency_code' => $cart->cart_currency_code,
-                                'value'         => (float) $cart->tax_total,
+                                'value'         => $this->smartButton->formatCurrencyValue((float) $cart->tax_total),
                             ],
 
                             'discount'   => [
                                 'currency_code' => $cart->cart_currency_code,
-                                'value'         => (float) $cart->discount_amount,
+                                'value'         => $this->smartButton->formatCurrencyValue( (float) $cart->discount_amount),
```

Comment:
Just remove one space from here,
`$this->smartButton->formatCurrencyValue( (float) $cart->discount_amount)`

`$this->smartButton->formatCurrencyValue((float) $cart->discount_amount)`